mapbook-iOS/AppContext/AppContext.swift

```diff
@@ -30,7 +30,9 @@ import ArcGIS
  */
 extension Notification.Name {
     
-    static let DownloadCompleted = Notification.Name("DownloadCompleted")
+    static let DownloadDidComplete = Notification.Name("DownloadDidComplete")
+    static let AppModeDidChange = Notification.Name("AppModeChanged")
+    static let PortalDidChange = Notification.Name("PortalDidChange")
```

Comment:
I think these three `static let` variables should start with a lower case, no?

Reply:
Oy i'm a dinosaur. Yes, the newer convention is to make notification names lower case.

Reply:
Wait a second there, T-Rex. I don't care about the strings, I care about the variables :) As far as I know, this would be fine:

``` Swift
static let downloadDidComplete = Notification.Name("DownloadDidComplete")
```